### PR TITLE
LPD-38942 Update file icon colors to new colors

### PR DESCRIPTION
**Pull request by pat270**
https://liferay.atlassian.net/browse/LPD-38942

We need https://github.com/liferay/clay/pull/5877 before we can merge this.

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss [hunk 1]
```diff
@@ -125,25 +125,25 @@ $toggle-card-on-icon-border-width: 0 !default;
...
-$file-icon-text-color-3: #af78ff !default;
-$file-icon-text-color-4: #ffb46e !default;
-$file-icon-text-color-5: #ff5f5f !default;
+$file-icon-text-color-0: $gray-600 !default;
+$file-icon-text-color-1: $gray-500 !default;
+$file-icon-text-color-2: $teal !default;
+$file-icon-text-color-3: $purple !default;
+$file-icon-text-color-4: $orange !default;
+$file-icon-text-color-5: $red !default;
 $file-icon-text-color-6: #4b9bff !default;
```
Comment (ethib137) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss:145]: @pat270 why is this the only color not being updated to a sass variable?

Comment (pat270) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss:145]: There was no match for that particular color in the spread sheet https://docs.google.com/spreadsheets/d/1SF0_nFyXVPbZCFrOyDt5sTe0vg-rt8it9AhdHKtTbac/edit?pli=1&gid=591426718#gid=591426718. $blue-l1 is similar.

Comment (drakonux) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss:145]: Hey, correct me if I am wrong, but the colors are:

image

Comment (ethib137) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss:145]: @pat270 can you test this and share a screenshot of the final result. You should be able to reproduce this uploading files of the types above to the document library. Thanks.

Comment (pat270) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss]: I updated the pr. I changed the wrong variables and made the background's all white. There is a bug where the correct `file-icon-color-#` class isn't being implemented. I had to manually change the classes for this screenshot.

file-icon-color

**Comment by pat270** [comment]
ci:test:sf

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss (hunk 1 above)
Comment (ethib137) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss]: @pat270 what's the bug? Can you create a ticket for it?

**Comment by ethib137** [comment]
ci:forward

**Pull request closed without merging**